ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/jboss/narayana/txframework/main/module.xml

```diff
@@ -37,7 +37,6 @@
         <!--WFLY-14219 Remove deprecated <module name="javax.api"/> -->
         <module name="javax.enterprise.api"/>
         <module name="org.jboss.xts" />
-        <module name="org.jboss.resteasy.resteasy-jaxrs" />
```

Comment:
I'm not sure about this change. The txframework works over the XTS and RTS-AT transaction submodules. There is the resteasy needed in case of some scenarios.
https://github.com/jbosstm/narayana/blob/master/txframework/src/main/java/org/jboss/narayana/txframework/impl/handlers/HandlerFactory.java#L97
Was that considered?

Reply:
Hmm.. ...you're right. I missed this some how. I'll update it to rely on resteasy-core.